omada-controller.json

```
@@ -1,172 +1,194 @@
+            "label": "Device discovery [e.g. 29810]",
+            "index": 9
+          },
+          "PORT_MANAGER_V1": {
+            "description": "Omada Controller and Omada Discovery Utility manage Omada devices running firmware fully adapted to Omada Controller v4.",
+            "label": "Device managing (v4) [e.g. 29811]",
+            "index": 10
+          },
+          "PORT_MANAGER_V2": {
+            "description": "Omada Controller and Omada Discovery Utility manage Omada devices running firmware fully adapted to Omada Controller v5.",
+            "label": "Device managing (v5) [e.g. 29814]",
+            "index": 11
+          },
+          "PORT_UPGRADE_V1": {
+            "description": "When upgrading the firmware for the Omada devices running firmware fully adapted to Omada Controller v4.",
+            "label": "Firmware upgrade (v4) [e.g. 29813]",
+            "index": 12
+          },
+          "PORT_TRANSFER_V2": {
+            "description": "Omada Controller receives Device Info and Packet Capture files from the Omada devices.",
+            "label": "Device info (v5) [e.g. 29815]",
+            "index": 13
+          },
+          "PORT_RTTY": {
+            "description": "Omada Controller establishes the remote control terminal session with the Omada devices.",
+            "label": "Remote control (v5) [e.g. 29816]",
+            "index": 14
+          },
+          "SHOW_SERVER_LOGS": {
+            "description": "Outputs Omada Controller logs to STDOUT at runtime.",
+            "label": "Show server logs [e.g. true]",
+            "index": 15
+          },
+          "SHOW_MONGODB_LOGS": {
+            "description": "Outputs MongoDB logs to STDOUT at runtime.",
+            "label": "Show MongoDB logs [e.g. false]",
+            "index": 16
+          },
+          "SSL_CERT_NAME": {
+            "description": "Name of the public certificate chain mounted to /cert.",
+            "label": "SSL certificate name [e.g. tls.crt]",
+            "index": 17
+          },
+          "SSL_KEY_NAME": {
+            "description": "Name of the private certificate mounted to /cert.",
+            "label": "SSL key name [e.g. tls.key]",
+            "index": 18
+          },
+          "TLS_1_11_ENABLED": {
+            "description": "Re-enables TLS 1.0 and 1.1 if set to true.",
+            "label": "Enable unsecure TLS versions [e.g. false]",
+            "index": 19
+          }
+        }
+      }
+    },
+    "description": "TP-Link Omada Controller. <p><strong>Note: in case of adoption failures, <a href='https://github.com/mbentley/docker-omada-controller/blob/master/KNOWN_ISSUES.md#devices-fail-to-adopt' target='_blank'>read this!</a></strong></p> <p>Based on a custom docker image: <a href='https://hub.docker.com/r/mbentley/omada-controller' target='_blank'>https://hub.docker.com/r/mbentley/omada-controller</a>, available for amd64 and arm64 architecture.</p>",
```

```suggestion
    "description": "TP-Link Omada Controller. <p><strong>Note: in case of adoption failures, <a href='https://github.com/mbentley/docker-omada-controller/blob/master/KNOWN_ISSUES.md#devices-fail-to-adopt' target='_blank'>read this!</a></strong></p> <p>Based on a <a href='https://hub.docker.com/r/mbentley/omada-controller' target='_blank'>custom docker image</a>, available for amd64 and arm64 architecture.</p>",
```

Moving towards hyperlinking the custom/standard image as opposed to the actual URL.